website/content/docs/job-specification/hcl2/functions/crypto/rsadecrypt.mdx

```diff
@@ -6,6 +6,12 @@ description: The rsadecrypt function decrypts an RSA-encrypted message.
 
 # `rsadecrypt` Function
 
+-> **Deprecated:** This function is deprecated and may be removed in the
+future. While it is possible to use safely, it encourages use of cryptographic
+primitives with [known weaknesses][attacks]. [Nomad Variables][nvars] and
+[HashiCorp Vault][vault] are the recommended ways to provide secrets to
+workloads.
```

Comment:
Should we leverage the newer MDX components for these callouts?

```suggestion
<Note title="Deprecated">
This function is deprecated and may be removed in the future.
</Note>

<!--
The whitespace inside the Warning tag is significant because it contains markdown
-->
<Warning>

While it is possible to use safely, it encourages use of cryptographic
primitives with [known weaknesses][attacks]. [Nomad Variables][nvars] and
[HashiCorp Vault][vault] are the recommended ways to provide secrets to
workloads.

</Warning>

```

Reply:
I love it! Thank you! I had to rewrite the links in HTML, but nbd.